### PR TITLE
OP-625 demo data cause NPE in admissions

### PR DESCRIPTION
**Pull request by mwithi**
See OP-625.

### CODE REVIEW

File: sql/load_demo_data.sql [hunk 1]
```diff
@@ -24,11 +24,11 @@ DROP TABLE IF EXISTS `admission`;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `admission` (
   `ADM_ID` int(11) NOT NULL AUTO_INCREMENT,
-  `ADM_IN` int(11) NOT NULL DEFAULT 0,
+  `ADM_IN` int(11) NOT NULL DEFAULT '0',
```
Comment (dbmalkovsky) [inline sql/load_demo_data.sql:27]: has the dump format changed?   `ADM_IN` is an `int` yet the default value looks like a `string`: '0' not as an integer zero as before.

Comment (mwithi) [reply sql/load_demo_data.sql:27]: I noticed that, I think it's fine, but I'm still investigating it...

Comment (mizzioisf) [reply sql/load_demo_data.sql:27]: mwhiti you should use mariadb if possible.

Comment (mwithi) [reply sql/load_demo_data.sql:27]: [here](https://stackoverflow.com/a/35514653) there is a good explanation, nothing to worry about I think... ​

Comment (dbmalkovsky) [reply sql/load_demo_data.sql:27]: IMHO it makes the project look less professional as someone reading the codebase is going to scratch their head and wonder what is going on....they shouldn't have to do research and understand that there is data conversion going on under the covers.   Again just my opinion.

Comment (mwithi) [reply sql/load_demo_data.sql:27]: IMHO `MySQL dump 10.13  Distrib 5.7.17, for Win64 (x86_64)` explain everything...

we'll use another client like mariadb (although I don't know if on Windows has same exact behavior)

Comment (mwithi) [reply sql/load_demo_data.sql:27]: Done again with MariaDB for Windows (taken from portable OH), the default values use same syntax

Comment (mwithi) [reply sql/load_demo_data.sql:27]: Can we merge this or should we stick only to linux-forged mysql dumps?

Comment (mizzioisf) [reply sql/load_demo_data.sql:27]: I think we can merge and get back on this after release; I will work on the demo data anyway.

**Pull request merged**